### PR TITLE
[C-2719] Clean up and document non v1 usage

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

* Clean up unused/easily replaceable non-v1 API calls
* Document with `//TODO(C-2719)` the rest of them

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Locally vs. staging
* Under the radar
* Profile pages
* Signup/upload (for sanity)
* Bunch of clicking around

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-2719

**Review by dylanjeffers — APPROVED** [review]
STELLAR fix, so cool you were able to do that so quick. Do we also want to go through the non-v1 endpoints in AudiusApi? i think all of them with "RootPath"?

File: packages/web/src/common/store/cache/tracks/sagas.js [hunk 1]
```diff
@@ -249,10 +249,7 @@ function* deleteTrackAsync(action) {
   const handle = yield select(getUserHandle)
 
   // Before deleting, check if the track is set as the artist pick & delete if so
-  const socials = yield call(
-    audiusBackendInstance.getCreatorSocialHandle,
-    handle
-  )
+  const socials = yield call(audiusBackendInstance.getSocialHandles, handle)
```
Comment (sliptype) [inline packages/web/src/common/store/cache/tracks/sagas.js:252]: For these js files I would just double check that the data is returned in the same format

**Review by sliptype — APPROVED** [review]
Let's goooo

File: packages/web/src/common/store/cache/tracks/sagas.js (hunk 1 above)
Comment (raymondjacobson) [reply packages/web/src/common/store/cache/tracks/sagas.js:252]: this fn is just renamed :) but yes get your point!!

**Pull request merged**